### PR TITLE
Slack integration

### PR DESCRIPTION
**Pull request by shivpatel779**
Hello @reneweteling

WIP - Added ' slack-ruby-client ' and sets a global default slack token for application

### CODE REVIEW

File: config/initializers/slack.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+require "slack"
+
+Slack.configure do |config|
+  config.token = ENV['SLACK_API_TOKEN']
```
Comment (reneweteling) [inline config/initializers/slack.rb:4]: :100 nice!

File: config/slack.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+default: &defaults
+  SLACK_API_TOKEN: "xoxp-68890467382-68899994631-68893956256-60171994ec"
+  # Your app's name
```
Comment (reneweteling) [inline config/slack.yml:3]: neeeeeeeehhh.... should be in .env or something, but ok

**Comment by reneweteling** [comment]
your pr got a bit poluted with the stuf you have done previously, the testing. Also the tests are failing in the CI

And what i like to do are some naming convenstions so start a PR with:
- wip ( when your working on it )
- revieuwme ( when you are done and the tests are green )
- mergeme (once the revieur finds its ok)

Besides from that, nice!

**Comment by shivpatel779** [comment]
Creating with new PR for slack integration branh

**Pull request closed without merging**